systest/cluster/nodes.go

```diff
@@ -872,14 +876,31 @@ func deployNode(
 		cmd = append(cmd, flag.Flag())
 	}
 
+	pvcName := fmt.Sprintf("data-%s", id)
+	pvc := corev1.PersistentVolumeClaim(pvcName, ctx.Namespace).
+		WithSpec(corev1.PersistentVolumeClaimSpec().
+			WithStorageClassName("premium-rwo").
```

Comment:
I'm not sure hardcoding storage class name is good here, it's very cloud-provider-specific.
I run systests on my k3s cluster sometimes, and the storage class I need to use is different there.
It would make sense not to add any storage class by default so that the default storage class is used,
and add an option to specify it, so it can be added to the environment variables in the CI job.